inst/rmarkdown/templates/jds/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,271 @@
+environment. For example,
+\begin{align*}
+  g_{\mu}(\phi) = 0, \\
+  g_{\mu}(X) = 1.
+\end{align*}
+
+
+An equation that is not referenced should not be labeled. The starred
+version of the \texttt{equation} and \texttt{align} are for this purpose.
+
+# Tables {#sec:tabs}
+
+We recommend \LaTeX\ package `booktabs` for professional
+looking tables. Its toprule and bottomrule are thicker than midrule.
+
+A professional table contains no vertical lines.
+
+Table \ref{tab:realdata} is an illustration.
+
+\begin{table}[tbp]
+  \caption{Analysis results for real data. Point
+    estimates (EST) from both two-piece method and marginal method are
+    reported. Standard error of point estimates are evaluated by
+    parametric bootstrap (SE).}%
+  \label{tab:realdata}
+  \centering
+  \begin{tabular}{llrrrr}
+    \toprule
+    \multicolumn{1}{l}{Season} & \multicolumn{1}{l}{Parameter}
+                               & \multicolumn{2}{c}{Two-piece method} & \multicolumn{2}{c}{Marginal method}                         \\
+    \cmidrule(r){3-4} \cmidrule(r){5-6}
+                               &                                      & EST                                 & SE    & EST   & SE    \\
+    \midrule
+    Summer                     & $\lambda_1$                          & 2.841                               & 0.459 & 1.090 & 0.280 \\
+                               & $\lambda_0$                          & 0.179                               & 0.014 & 0.158 & 0.015 \\
+                               & $\sigma$                             & 1.335                               & 0.106 & 0.999 & 0.104 \\
+                               & $\sigma_\epsilon (\times 10^{-2})$   & 1.854                               & 0.087 & 1.879 & 0.078 \\
+    Winter                     & $\lambda_1$                          & 6.225                               & 0.825 & 4.720 & 0.711 \\
+                               & $\lambda_0$                          & 0.118                               & 0.010 & 0.114 & 0.009 \\
+                               & $\sigma$                             & 1.506                               & 0.095 & 1.454 & 0.089 \\
+                               & $\sigma_\epsilon (\times 10^{-2})$   & 0.908                               & 0.036 & 0.934 & 0.043 \\
+    \bottomrule
+  \end{tabular}
+\end{table}
+
+
+
+
+# Figures {#sec:figs}
+
+Vector graphics do not lose clarity when being scaled. Make your
+figure in pdf format when you first generate it and keep in mind its
+sizes in the article to avoid over-scaling. Do not simply convert a
+jpeg or png image to a pdf.
+
+# Code {#sec:code}
+
+The document class \texttt{jds} provides several commands to decorate
+\begin{itemize}
+  \item inline code, such as \code{print("Hello world!")};
+  \item programming language, such as \proglang{R}, \proglang{Python}, and
+        \proglang{C++};
+  \item software package, such as \pkg{stats}, \pkg{utils}.
+\end{itemize}
```

Comment:
What about converting to Markdown here and not just insert the LaTeX syntax ? 

Should be somethink like 

```suggestion
The document class `jds` provides several commands to decorate

- inline code with `\code{}`, such as \code{print("Hello world!")};
- programming language with `\proglang{}`, such as \proglang{R}, \proglang{Python}, and
        \proglang{C++};
- software package with `\pkg`, such as \pkg{stats}, \pkg{utils}.
```